src/framework/piral-core/src/modules/element.ts

```diff
@@ -10,12 +10,19 @@ import {
   extensionName,
   slotName,
   isSame,
+  contentName,
 } from '../utils';
 
 export interface Updatable {
   (newProps: any): void;
 }
 
+declare global {
+  interface Window {
+    assignContent(cid: string, content: any): void;
```

Comment:
I think that it might be really helpful if there was a JSDoc comment here with an example about how this can be used together with the custom element below since this is a paired API. 

The more realistic/real-life like, the better. 😄

Reply:
Cool! Yeah, I made that comment mainly because I didn't really get how this is used / what for. Another comment may help people reading that code in the future, so it's great to have!